etc/generate-envoy-config/envoy.jsonnet

```diff
@@ -0,0 +1,196 @@
+// Services that we want to proxy; where they are and how to route them.
+local services = {
+  'pachd-grpc': {
+    internal_port: 1650,
+    external_port: 30650,
+    grpc: true,
+    service: 'pachd-proxy-backend',
+    routes: [
+      {
+        match: {
+          grpc: {},
+          prefix: '/',
+        },
+        route: {
+          cluster: 'pachd-grpc',
+          timeout: '604800s',
+        },
+      },
+    ],
+    health_check: {
```

Comment:
where's the reference to these health check values?

Reply:
Basically if there's more than one pachd, and one of them isn't working, we can avoid sending traffic to that one.  Otherwise, if we look at the error logs we can see things like "no healthy upstream" and know that, at the time, pachd wasn't being responsive.  (Will be nice in debug dumps, for example.)